WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -1311,7 +1311,7 @@ private void setSite(Intent data) {
 
     @Override
     @SuppressWarnings("deprecation")
-    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+    protected void onActivityResult(int requestCode, int resultCode, @Nullable Intent data) {
```

Comment:
> Reseting it makes sense, since if there was a need for this to be public, the build would fail. 👍

Exactly, and again, I am going for consistency here and making this safe minor changes seems easy enough for the better they produce, not having us to worry again about why the `protected` was/is used. 👍